Composer/packages/client/src/components/ProjectTree/treeItem.tsx

```diff
@@ -38,6 +38,8 @@ const content = css`
   outline: none;
   display: flex;
   align-items: center;
+  justify-items: middle;
```

Comment:
perhaps you meant 'center'
https://developer.mozilla.org/en-US/docs/Web/CSS/justify-items